ferrocene/doc/qualification-plan/src/standards/iec-61508.rst

```diff
@@ -8,127 +8,169 @@ IEC 61508 requirements
    :header-rows: 1
 
    * - Clause
-     - Document chapter(s)
+     - ``rustc``
+     - ``rustfmt``
 
    * - 7.4.4.1
      - N/A
+     - N/A
 
    * - 7.4.4.2
      - N/A
+     - N/A
 
    * - 7.4.4.3
      - This qualification is the justification
+     - This qualification is the justification
```

Comment:
am thinking 7.4.4.2 being N/A makes this N/A (but am maybe missing something)

Reply:
I think this is fine. 7.4.4.2 is meant for our customers (they need to justify why they chose their tools), while 7.4.4.3 is the beginning of the part we cover ("providing such justification").